solana-library/minimum_balance.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: Apache-2.0
+
+// Disclaimer: This library provides functions for working with storage rent. Although it is production ready,
+// it has not been audited for security, so use it at your own risk.
+
+// This is the Solidity version of the rust module rent:
+// https://github.com/solana-labs/solana/blob/master/sdk/program/src/rent.rs
+// As rent is currently not implemented on Solana, only the minimum balance is required.
+
+/// Default rental rate in lamports/byte-year.
+///
+/// This calculation is based on:
+/// - 10^9 lamports per SOL
+/// - $1 per SOL
+/// - $0.01 per megabyte day
+/// - $3.65 per megabyte year
+uint64 constant DEFAULT_LAMPORTS_PER_BYTE_YEAR = 1_000_000_000 / 100 * 365 / (1024 * 1024);
+
+/// Default amount of time (in years) the balance has to include rent for the
+/// account to be rent exempt.
```

Comment:
This description is a bit unclear.

Reply:
I've reworded it. Note this is all academy since rent is not implemented yet.